src/test/scala/com/microsoft/hyperspace/index/rules/HyperspaceRuleTestSuite.scala

```diff
@@ -51,7 +51,10 @@ trait HyperspaceRuleTestSuite extends HyperspaceSuite {
                 .Columns(indexCols.map(_.name), includedCols.map(_.name)),
               IndexLogEntry.schemaString(schemaFromAttributes(indexCols ++ includedCols: _*)),
               10)),
-          Content(getIndexDataFilesPath(name).toUri.toString, Seq()),
+          Content(
+            Directory(
+              getIndexDataFilesPath(name).toUri.toString,
+              Seq(FileInfo("v__=0/f1.parquet", 10, 10)))),
```

Comment:
having "/" is inside `FileInfo` is not semantically correct after this PR right?

Reply:
thanks @imback82 , fixed